src/Applications/LDAS_App/ldas_setup

```diff
@@ -1531,26 +1504,27 @@ def _printRmInputKeys(rqdRmInpKeys, optRmInpKeys):
     print '# REQUIRED inputs'
     print '#'
     print '# NOTE:'
-    print '# - rm_name    = resource manager name'
-    print '#                  [At NCCS: Use "SLURM"; PBS is no longer used.'
-    print '# - account    = computational project number'
+    print '# - account      = computational project number'
     print '#                  [At NCCS: Use command "getsponsor" to see available account number(s).]'  
-    print '# - walltime   = walltime requested; format is HH:MM:SS (hours/minutes/seconds)'
-    print '# - ntasks     = number of processors requested'
+    print '# - walltime     = walltime requested; format is HH:MM:SS (hours/minutes/seconds)'
+    print '# - ntasks_model = number of processors requested for the model (output server is not included)'
+    print '# - ntasks-per-node  = number of tasks per node(  >28 , sky will be allocated, <=28 sky or hasw'
     print '#'
     for key in rqdRmInpKeys:
-        print key.ljust(23), ':'
+        print key + ':'
     print
     print '#'
     print '# OPTIONAL inputs'
     print '#'
     print '# NOTE:'
     print '# - job_name   = name of experiment; default is "exp_id"'
-    print '# - constraint = type of processors requested; default is no constraint.'
     print '# - qos        = quality-of-service; do not specify by default; specify "debug" for faster but limited service.'
+    print '# - oserver_nodes    = number of nodes for oserver ( default is 1 ), set it to 0 expicilty if no output server needed'
```

Comment:
Is the "default is 1" here in conflict with line 471, where it looks like the default is 0?
(Remember to fix the typo "explicitly")

Reply:
Yes, it should be 0